tests.py

```diff
@@ -141,6 +141,7 @@ def run_tests():
         },
         'POST',
     )
+    """
```

Comment:
Should these be commented out or deleted? If commented out maybe we should put a message as to why they are?